lib/convertkit/resources/subscribers.rb

```diff
@@ -0,0 +1,67 @@
+module ConvertKit
+  module Resources
+    class Subscribers
+      PATH = 'subscribers'.freeze
+
+      def initialize(client)
+        @client = client
+      end
+
+      # Returns a list of subscribers for the account.
+      # See https://developers.convertkit.com/#list-subscribers for details
+      # @param [Hash] options
+      # @option options [Integer] :page The page of results to return.
+      # @option options [Integer] :from Filter subscribers added on or after date
+      # @option options [Integer] :to Filter subscribers added on or before date
+      # @option options [String] :updated_from Filter subscribers updated after date
+      # @option options [String] :updated_to Filter subscribers updated before date
+      # @option options [String] :sort_order Sort order for results(`asc` or `desc`)
+      # @option options [String] :sort_field Field to sort results by
+      # @option options [String] :email_address Search subscribers by email address
+      def get_subscribers(options = {})
```

Comment:
For the method names, I'm wondering if we can follow the naming for the endpoint and call this just `list` instead of `get_subscribers`.
That way the call pattern would become `subscribers.list` vs `subscribers.get_subscribers`

Reply:
I just checked and Stripe also follows something similar where their api for any one object for list is called `list`
So `Stripe::Coupon.list` or `Stripe::Event.list`